src/containers/my-certificates/filled-certificates/filled-certificates.js

```diff
@@ -1,49 +1,48 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
 import { useTranslation } from 'react-i18next';
-import Button from '@material-ui/core/Button';
 
-import { Link } from 'react-router-dom';
-import CertificateTable from '../certificate-table';
-import { useStyles } from './filled-certificates.styles';
-import { Loader } from '../../../components/loader/loader';
+import Button from '@material-ui/core/Button';
+import Modal from '../../../components/modal';
 import routes from '../../../configs/routes';
+import { useStyles } from './filled-certificates.styles';
+import CertificateTable from '../certificate-table';
 
 const FilledCertificates = ({ items }) => {
+  const [modalVisibility, setModalVisibility] = useState(false);
   const styles = useStyles();
   const { t } = useTranslation();
+  const modalHandler = () => {
```

Comment:
Work with naming, for me it looks like 'openModal', and the next one should be 'closeModal', but you can create one handler that will receive true or false, or work with setModalVisibility directly. Cause it is the only thing that this method is working with.

Reply:
Fixed. Change for: 

 const openModal = () => {
    !modalVisibility ? setModalVisibility(true) : setModalVisibility(false);
  };